certified-connectors/BulkSMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,126 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "BulkSMS",
+    "description": "BulkSMS Text Message Connector",
+    "contact":{
+      "name": "BulkSMS Support",
+      "url": "https://www.bulksms.com/support/",
+      "email":"support@bulksms.com"
+    }
+  },
+  "host": "api.bulksms.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": ["application/json"],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v1/messages": {
+      "post": {
+        "summary": "Send SMS Message",
+        "description": "Send an SMS Text message",
+        "operationId": "SendSmsMessage",
+        "parameters": [
+          {
+            "name": "auto-unicode",
+            "default": true,
+            "in": "query",
+            "type": "boolean",
+            "required": true,
+            "description": "Convert the message to Unicode if required",
+            "x-ms-summary": "Auto-Unicode",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "from": {
+                  "type": "string",
+                  "description": "from"
+                },
+                "to": {
+                  "type": "string",
+                  "description": "to",
+                  "title": ""
+                },
+                "body": {
+                  "type": "string",
+                  "description": "body",
+                  "title": ""
+                },
+                "userSuppliedId": {
+                  "type": "string",
+                  "description": "userSuppliedId",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "BLKTM.GWPF.01.00.00"
```

Comment:
Thanks @Amjed-Ayoub 
This connector sends SMS messages via our JSON REST API. We need this UserSuppliedId so that we can identify messages that have been sent via this connector and the also the version of the connector. This enables us to identify problems with the connector and thereby provide better support to our customers. This identifier will also be useful for later iterations when we start returning reply messages and status reports.

`BLKTM` indicates and integration
`GWPF` indicates 'Powerflow'
`01.00.00` indicates the version (major, minor, revision)